src/api/Nhs.Appointments.Api/Notifications/template-bookingmade-rsv-email.txt

```diff
@@ -10,20 +10,27 @@ Template Body:
 
 Hello ((firstName)),
 
-This is a reminder for your upcoming RSV (respiratory syncytial virus) vaccination appointment.
+Thank you for booking your flu vaccination appointment.
```

Comment:
Should this say RSV? The template name is includes rsv so I'm thinking it shouldn't say flu?